driver-core/src/main/com/mongodb/ClientSessionOptions.java

```diff
@@ -49,6 +50,20 @@ public Boolean isCausallyConsistent() {
         return causallyConsistent;
     }
 
+    /**
+     * Whether read operations using this session should all share the same snapshot.
+     *
+     * @return whether read operations using this session should all share the same snapshot. A null value indicates to use the global
+     * default, which is currently false.
```

Comment:
There is precedent in the drivers for changing the default from one release to another.  In particular, we changed the default for retryWrites from false to true.  But in this case it would change semantics to change the default, so I don't think we would ever change the default.

Reply:
We actually have the same language just above for the isCausallyConsistent method.

Reply:
Done